src/components/layout.js

```diff
@@ -25,6 +26,10 @@ const Layout = ({ children }) => (
     `}
     render={data => (
       <>
+        <Helmet>
+          <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/655912/7241412/css/fonts.css" />
```

Comment:
I know this PR is a WIP, but I was checking it out to see if I could branch some other work off of it. I noticed that `Whitney` does not render correctly in the browser for me. I started to look into it, but didn't find an answer (yet 😃).

<img width="697" alt="whitney" src="https://user-images.githubusercontent.com/12678977/58351909-13fb1080-7e38-11e9-9d4d-7c12655ba1f9.png">